torch/nn/modules/instancenorm.py

```diff
@@ -44,7 +44,7 @@ class InstanceNorm1d(_InstanceNorm):
         This :attr:`momentum` argument is different from one used in optimizer
         classes and the conventional notion of momentum. Mathematically, the
         update rule for running statistics here is
-        :math:`\hat{x}_\text{new} = (1 - \text{momentum}) \hat{x} + \text{momemtum} x_t`,
+        :math:`\hat{x}_\text{new} = (1 - \text{momentum}) \hat{x} + (\text{momemtum}) x_t`,
```

Comment:
I don't think there should be parentheses here.

Reply:
The words get fused otherwise. I think I can change it using a space. Thanks!